### PR TITLE
Update some files (pkg.json, .gitignore ,  license)

### PR DESCRIPTION
**Pull request by heithemmoumni**
- [ ] Updated Unreleased section in [CHANGELOG](https://github.com/reviewdog/reviewdog/blob/master/CHANGELOG.md) or it's not notable changes.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,7 @@
+.idea
+.DS_Store
 .drone.sec.yaml
 vendor/
 node_modules/
 dist/
+package-lock.json
```
Comment (haya14busa) [inline .gitignore:7]: Can you remove it?

Comment (heithemmoumni) [reply .gitignore:7]: ok

File: LICENSE [hunk 2]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2016 haya14busa
+Copyright (c) 2016 - present haya14busa
```
Comment (haya14busa) [inline LICENSE:3]: In my understanding, we don't need to update copyright of year and I'm not sure `- present` is actually valid.
Can you revert the change?

**Comment by haya14busa** [comment]
Please re-open when you incorporate the comment.
